<launch>
    <node pkg="roslaunch_axserver" type="roslaunch_server.py" name="record_roslaunch_axserver" output="screen"/>
    <node pkg="mongodb_openni_compression" type="record_server.py" name="record_camera" output="screen"/>
</launch>
